src/NServiceBus.StructureMap/StructureMapObjectBuilder.cs

```diff
@@ -181,7 +181,7 @@ static ILifecycle GetLifecycleFrom(DependencyLifecycle dependencyLifecycle)
             case DependencyLifecycle.SingleInstance:
                 return new SingletonLifecycle();
             case DependencyLifecycle.InstancePerUnitOfWork:
-                return null;//null means the default lifecycle which is transient
+                return new ContainerLifecycle();//null means the default lifecycle which is transient
```

Comment:
Yeah I was probably too fast to hit the merge button. My brain removed that comment.